content/docs/developing-operators.md

```diff
@@ -71,19 +76,27 @@ spec:
 This is a pretty normal Kubernetes YAML file defining a deployment. However, you can already see the KUDO templating language in action on the line referencing `.Params.Replicas`. This will get substituted during installation by merging what is in `params.yaml` and overrides defined before install. So let’s define the last missing piece, `params.yaml`.
 
 ```yaml
-replicas:
-  description: Number of replicas that should be run as part of the deployment
-  default: 2
+apiVersion: kudo.dev/v1beta1
+parameters:
+  - name: replicas
+    description: Number of replicas that should be run as part of the deployment
+    default: 2
 ```
 
 Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance` as described in [concepts](concepts.md).
 
 **Note:** If you want to install the result of the following steps with doing them manually, you can clone the KUDO repository and run the example from there:
 
 ```bash
-git clone https://github.com/kudobuilder/kudo.git
-cd kudo
-kubectl kudo install ./config/samples/first-operator
+git clone https://github.com/kudobuilder/operators.git
+cd operators
+kubectl kudo install ./repository/first-operator/operator/
+```
+
+or you can just simply do
+
+```bash
+kubectl kudo install first-operator
```

Comment:
$ kubectl kudo install first-operator
Error: failed to resolve package CRDs for operator: first-operator: getting first-operator in index file: no operator found for: first-operator